test/backend/routes.spec.js

```diff
@@ -269,9 +271,10 @@ describe('Authentication - ', () => {
             GET('settings')
```

Comment:
In order for errors inside the `.then` of this functions to bubble up to the `.catch` on very outside, you'll need to `return` this `GET` (I believe)